static/js/calendar.js

```diff
@@ -97,6 +97,9 @@ function make_cal(name) {
                         return '<span class="tui-full-calendar-dayname-name"> ' + parts[1] + '/' + parts[2] + '</span>&nbsp;&nbsp;<span class="tui-full-calendar-dayname-name">' + model.dayName + '</span>';
                     },
                 },
+                theme: {
+                    'common.holiday.color': '#333'
```

Comment:
this could probably be a config variable? and use `{{ config.calendar.colors.xxxx }}`
Just a note here -- we can address this in future as enhancements.